src/main/webapp/partials/instructorHelpSections.jsp

```diff
@@ -0,0 +1,65 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="sections">Sections</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="sectionsWhatIsItMeantFor">
+                    <h3>1. What are sections meant for?</h3>
+                </a>
+            </span>
+            <p>
+                Sections are meant for organizing students in courses with significantly large numbers of students. It is mandatory for courses with more than 100 students to organize a student into a section.
```

Comment:
This one hasn't been fixed yet: `... to organize students into sections.` (very last part of last sentence)